CHANGELOG.next.asciidoc

```diff
@@ -201,6 +201,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add specific date processor to convert timezones so same pipeline can be used when convert_timezone is enabled or disabled. {pull}12253[12253]
 - Add MSSQL module {pull}12079[12079]
 - Add full ISO8601 date parsing support for system/syslog module. {pull}12568[12568]
+- Add full ISO8601 date parsing support for system/auth module. {pull}12578[12579]
```

Comment:
I didn't think on it on #12568, but should we remove the entry added for #12543 now that we have these entries?

Reply:
+1 on combining some changelogs